content/blog/2019/r-version-support/index.md

```diff
@@ -15,36 +15,24 @@ rmd_hash: efe6168c59ef5121
 
 ---
 
-## Which versions of R do tidyverse packages support?
+Our general policy is to support the **current version**, the **devel version**, and **four previous versions** of R. This policy applies to all packages that we support, i.e. as well as all tidyverse packages, it also applies to the infrastructure packages that we maintain in [r-lib](https://github.com/r-lib).
 
-Though package-specific minimum versions of R are given in the `Depends` fields of individual `DESCRIPTION` files, our general policy is to support the **current version**, the **devel version**, and **four previous versions** of R.
+This, the *official* minimum supported versions of R for the tidyverse are as described here:
```

Comment:
I feel like the comma is weird. Should it go away? Get a matching friend? Or do you mean "Thus"?